README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/daostack/arc.svg?branch=master)](https://travis-ci.org/daostack/arc)
+[![Build Status](https://travis-ci.org/@daostack/arc.svg?branch=master)](https://travis-ci.org/daostack/arc)
```

Comment:
You are writing you changed the npm bedge, but you are changing the travis bedge